app/wrappers/device/device_info.ts

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
Our components library has `DeviceProvider` implementation. Don't need to create duplication.